src/__tests__/Groups/Group.test.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import '@testing-library/jest-dom';
```

Comment:
- [ ] This test fails and throws the following error. But once I remove this line of code, it passes! The same goes for all the other test cases you have
```
 FAIL  src/__tests__/Header/Header.test.js
  ● Test suite failed to run

    Cannot find module '@testing-library/jest-dom' from 'src/__tests__/Header/Header.test.js'

      1 | import React from 'react';
    > 2 | import '@testing-library/jest-dom';
        | ^
      3 | import { Provider } from 'react-redux';
      4 | import renderer from 'react-test-renderer';
      5 | import {

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:306:11)      at Object.<anonymous> (src/__tests__/Header/Header.test.js:2:1)
```